Pyrado/scripts/training/qq-su_reps_sprl_cov_only.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2020, Fabio Muratore, Honda Research Institute Europe GmbH, and
```

Comment:
please only push **1** SPRL training file per env and subroutine, and exlude all others